nix/t1/release/doc.typ

```diff
@@ -37,20 +37,49 @@
 
 Scalar core cannot access Vector DDR/SRAM, for, users need to access corresponding memory banks via vector load store instructions.
 
-== How to use the Docker image
+== Installing Docker for Debian-based systems.
+
+Refer to the Official Installation Guide for Docker:
+
+https://docs.docker.com/engine/install/
+
+== How to build the docker image locally
+
+Build the docker image using nix, copy the docker image to “/tmp”, hash the image and then load the docker image into docker
 
 #show raw.where(lang: "t1-docker"): it => {
   raw(lang: "bash", it.text.replace("${config}", config.name))
-}
+}  
+```t1-docker
+closure="$(nix build -L '.#t1.${config}.release.docker-layers.final-image' --no-link --print-out-paths)"
+echo "path: $closure"
+sudo cp "$closure/image.tar" /tmp/t1-${config}-image.tar
+echo "cache-key-${config}=$(nix hash file --base32 $closure/image.tar)"
```

Comment:
What is this for? You can't just copy-pasting CI workflow without knowing its usage.

Reply:
I copy-pasted these to ensure consistency with the CI workflow. But you're right, that's irrelevant if the doc is meant to be only for downstream docker users. should I keep it only for the downstream docker users or make a separate section in the document for T1 developers too, WDYT?

And also what about the doc itself being built by nix, anyone accessing the doc would still need nix.

Reply:
No, this is irrelevant because you are echoing nonsense contents.

Reply:
This document is going to be relased as single individual PDF, which should be accessible at repository release page. Nix usage should be documented in readme.